model/database/registry.yaml

```diff
@@ -106,6 +106,28 @@ groups:
             'INSERT shipping_details SELECT orders',
             'get user by id',
           ]
+      - id: db.stored_procedure.name
```

Comment:
I think redis functions/scripts would fit into this category?
 
https://redis.io/docs/latest/commands/fcall/
https://redis.io/docs/latest/commands/evalsha/